cmd/estuary-shuttle/main.go

```diff
@@ -424,12 +433,14 @@ func (d *Shuttle) RunRpcConnection() error {
 		conn, err := d.dialConn()
 		if err != nil {
 			log.Errorf("failed to dial estuary rpc endpoint: %s", err)
-			time.Sleep(time.Second * 10)
+			time.Sleep(backoffTimer.NextBackOff())
 			continue
 		}
 
 		if err := d.runRpc(conn); err != nil {
 			log.Errorf("rpc routine exited with an error: %s", err)
+			backoffTimer.Reset()
+			// TODO Do we still need to wait here?
```

Comment:
We should at least sleep for the default backoff duration, ten seconds is definitely excessive at this point